### DISCUSSION TITLE
Load AR model below 100% of its scale

### DISCUSSION THREAD

**Discussion by jusuff415** [google/model-viewer#3422] [Q&A]
Hey! 

I have a problem with my USDZ model when I load it using the AR function of my Iphone. Its loading with a size of 100% but that is way too big and I always have to scale it down. I already tried reducing the size of the USDZ model but that doesnt really help. So my question is if there is any way to load the model on lets say 2% of its size?

I use it on my personal website here: 
https://playwithmyhead.com/

**Comment by elalish** [comment]
We have a feature for that for glTF, but we can't read USDZ, so you're on your own. Reducing the scale of the USDZ is the right approach; if that's not working it sounds like a bug in the exporter?
